<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>711</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>Input data</string>
    </property>
    <widget class="QTextEdit" name="inputText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>691</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCompile">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>290</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Compile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClear">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>290</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenFile">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>290</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open file</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>711</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>Output Data</string>
    </property>
    <widget class="QPushButton" name="btnSave">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>290</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save to file</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="outputText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>691</width>
       <height>261</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClear_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>290</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>701</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>735</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>inputText</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>572</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClear_2</sender>
   <signal>clicked()</signal>
   <receiver>outputText</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>689</y>
    </hint>
    <hint type="destinationlabel">
     <x>589</x>
     <y>656</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
